Matatika/matatika-docs
# Release 04_10_2021

- add 'latest' and 'verify' links
- dataset data example
- job status reference docs
- verify pipeline request
- quick start guide for running your pipeline locally
- data stores resource docs
- update job logs query examples 'text/plain' as default, 'application/stream+json', and 'application/x-ndjson'
- stop job endpoint docs
- and tons more great documentation improvements
